doc/commands/clone.md

```diff
@@ -39,7 +39,6 @@ a TFS source tree and fetch all the changesets
 								 initialize the merged branches
 								 * all: Manage merged changesets and initialize
 								 all the branches during the clone
-		  --batch-size=VALUE          Size of a the batch of tfs changesets fetched (-1 for all in one batch)
```

Comment:
We must keep this line (surely with the same fix you did in the .cs file). Because the options of the `fetch` command are also accessible from the `clone` command.

If you did this fix, I merge it ;)